src/component/plane/Avatar.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { RiFacebookCircleLine, RiInstagramLine } from 'react-icons/ri';
+import { TiSocialTwitterCircular } from 'react-icons/ti';
+import { Link } from 'react-router-dom';
+import styles from './avatar.module.scss';
+
+const Avatar = ({ details }) => (
+  <div className={styles.mainContainer}>
+    <Link style={{ width: 'inherit', display: 'block' }} to={`plane/${details.id}`}>
+      <img style={{ width: 'inherit' }} src={details.picture} alt="plane" />
+    </Link>
+    <div style={{
+      display: 'flex', flexDirection: 'column', justifyContent: 'center', alignItems: 'center',
+    }}
```

Comment:
- [ ] If you can use classes for styling here, it would be nice, since it's not a good practice to use the `style` attribute.